pkg/build/webhook/generic/generic_test.go

```diff
@@ -39,7 +39,7 @@ func GivenRequestWithPayload(t *testing.T, filename string) *http.Request {
 }
 
 func GivenRequestWithRefsPayload(t *testing.T) *http.Request {
-	data, err := ioutil.ReadFile("fixtures/post-receive-git.json")
+	data, err := ioutil.ReadFile("../../../../test/fixtures/webhook/generic/post-receive-git.json")
```

Comment:
define a helper in util for "FixturePath(segments ...string)"

On Tue, Sep 22, 2015 at 10:20 AM, Michal Fojtik <notifications@github.com>
wrote:

> In pkg/build/webhook/generic/generic_test.go
> <https://github.com/openshift/origin/pull/4742#discussion_r40091535>:
>
> > @@ -39,7 +39,7 @@ func GivenRequestWithPayload(t *testing.T, filename string) *http.Request {
> >  }
> >
> >  func GivenRequestWithRefsPayload(t *testing.T) *http.Request {
> > -	data, err := ioutil.ReadFile("fixtures/post-receive-git.json")
> > +	data, err := ioutil.ReadFile("../../../../test/fixtures/webhook/generic/post-receive-git.json")
>
> this is pretty ugly ;-) Can't we have some helper that will hide all these
> dots here? Something that knows where is the base dir (origin/)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4742/files#r40091535>.
>